api/v3/Payment.php

```diff
@@ -324,6 +333,15 @@ function _civicrm_api3_payment_get_spec(&$params) {
       'title' => 'Transaction ID',
       'type' => CRM_Utils_Type::T_STRING,
     ],
+    'trxn_date' => [
+      'title' => ts('Payment Date'),
+      'type' => CRM_Utils_Type::T_TIMESTAMP,
+    ],
+    'financial_trxn_id' => [
+      'title' => ts('Payment ID'),
+      'type' => CRM_Utils_Type::T_INT,
+      'api.aliases' => ['payment_id', 'id'],
+    ],
```

Comment:
@seamuslee001 What's the reason for exposing this - isn't it the same as `trxn_id` and doesn't this just add confusion?

Reply:
@mattwire this is already in 5.20

Reply:
@mattwire no - it's not. It's actually the same as 'financial_trxn_id' - ie to get a payment by id you need to pass in financial_trxn_id & if you pass in trxn_id  you get something different.

The entity should be gettable by id & making this change makes it so